src/routes/u/[playerId].svelte

```diff
@@ -46,6 +46,12 @@
 
    import type { LeaderboardPlayer, Player, PlayerScore, PlayerScoreCollection } from '$lib/models/PlayerData';
    import type { CanResetCountryData } from '$lib/models/CanResetCountryData';
+   import Presence from '$lib/components/player/presence.svelte';
+
+   async function loadPresenceComponent() {
```

Comment:
?

Reply:
whoops, this is 100% dead code i forgot to remove